app/assets/javascripts/govuk_publishing_components/components/step-by-step-nav.js

```diff
@@ -114,7 +126,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     var data = []
 
     for (var i = 0; i < this.$module.steps.length; i++) {
-      var stepView = new this.StepView(this.$module.steps[i], this.$module.actions.showText, this.$module.actions.hideText)
+      var stepView = new this.StepView(this.$module.steps[i], this.$module.actions.showText, this.$module.actions.hideText, this.$module.upChevronSvg, this.$module.downChevronSvg)
```

Comment:
These calls are starting to look shabby – I suggest we refactor to reduce it to `StepView(this.$module)` and call elements down the tree within the `StepView` function definition as needed.

Reply:
I've gone for a similar suggestion but kept the initial element parameter as the way this is passed differs from declaration to declaration and amending the code to cleanly meet just passing in `this.$module` each time would add extra verbosity. Good idea all the same, nice and tidy.